packages/core/src/entity/constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum PostMessageMethods {
+  REQUEST_ENTITIES = 'REQUEST_ENTITIES',
+  REQUESTED_ENTITIES = 'REQUESTED_ENTITIES',
+}
```

Comment:
Looks like this same enum is defined in `packages/core/src/constants.ts` as well